cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/VmCost.java

```diff
@@ -57,7 +56,11 @@ public Vm getVm() {
      * @return getVmMemoryCost
      */
     public double getVmMemoryCost() {
```

Comment:
the prefix "Vm" in methods such as "getVmMemoryCost" and "getVmBwCost" is redundant, since they are inside the VmCost class. Sometimes it is required to make things as clear as possible, but such kind of redundancy must be avoided. Thus the methods should be renamed to getMemoryCost, getBwCost, etc.